pkgdb2/lib/__init__.py

```diff
@@ -1712,13 +1712,25 @@ def add_new_branch_request(session, pkg_name, clt_to, user):
     if clt_to.name == 'Fedora EPEL':
         _validate_pkg(session, clt_to.version, package.name)
 
-    action = model.AdminAction(
+    action = model.AdminAction.search(
```

Comment:
It's still a list.

I saw ``action.pop()`` and thought "wtf.  An ``AdminAction`` has a ``.pop()`` method?"